USGS-VIZLAB/what-is-drought
# Clean up components

This is a small PR to clean up a few of the components after the move to `Vue 3`:
* Run lint and fix errors
* Fix styling of USWDS banner
* Update 404 page to USWDS template
* Fix styling of USGS footer (margins/padding)
* Update prefooter code links padding to match USGS footer

### To test
* pull down changes
* temporarily edit `App.vue` so that USWDS banner shows ( `<HeaderUSWDSBanner v-if="typeOfEnv === '-test build-'" />`) and window size component is hidden (`<WindowSize v-if="typeOfEnv !== '-test build-'" />`)
* run `npm run dev`
* check that USWDS banner, USGS footer, and pre footer code links look good
* check that 404 page looks good by adding `/404` to end of url
